client/app/startup/SearchPageApp.js

```diff
@@ -27,18 +28,36 @@ export default (props) => {
 
   const bootstrappedData = TransitImmutableConverter.fromJSON(props.data);
 
-  const listingsToMap = (listings) =>
+  const listingWithAuthor = (listing, profiles) => {
+    const author = profiles.get(listing.authorId);
+    return listing.set('author', author);
```

Comment:
Should we just store the Listing to Redux store normalised, without the Author attached, and then attach the Author only in `mapStateToProps`? Let's plan.